packages/devtools_app/lib/src/inspector/flutter/layout_tab.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../inspector/inspector_text_styles.dart' as inspector_text_styles;
+import '../diagnostics_node.dart';
+import '../inspector_tree.dart';
+
+class InspectorDetailsTabController extends StatelessWidget {
+  const InspectorDetailsTabController({this.detailsTree, Key key})
+    : super(key: key);
+
+  final Widget detailsTree;
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 2,
+      child: Column(
+        children: <Widget>[
+          const TabBar(
+            tabs: [
+              Tab(text: 'Details Tree'),
+              Tab(text: 'Layout Details'),
+            ],
+          ),
+          Expanded(
+            child: TabBarView(
+              children: [
+                detailsTree,
+                Container(),
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+// TODO(albertusangga): Remove this linter ignore
+// ignore: must_be_immutable
+class StoryOfYourFlexWidget extends StatelessWidget {
+  StoryOfYourFlexWidget(this.node, {
+    Key key,
+  }) : super(key: key) {
+    properties = FlexProperties.fromJson(node.diagnostic.flexDetails);
+  }
+
+  final InspectorTreeNode node;
+
+  // Deserialized information about Flex elements
+  FlexProperties properties;
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> children = [
+      for (RemoteDiagnosticsNode child in node.diagnostic.childrenNow)
+        Expanded(
+          flex: 1,
+          child: Container(
+            decoration: BoxDecoration(
+              color: Theme
+                .of(context)
+                .backgroundColor,
+              border: Border.all(
+                color: Theme
+                  .of(context)
+                  .primaryColor,
+                width: 1.0,
+              ),
+              boxShadow: [
+                BoxShadow(
+                  color: Theme
+                    .of(context)
+                    .primaryColor,
+                  offset: Offset.zero,
+                  blurRadius: 10.0,
+                )
+              ],
+            ),
+            child: Center(
+              child: Text(child.description),
+            )),
+        )
+    ];
+    final Widget flexWidget = properties.type == Row
+      ? Row(children: children)
+      : Column(children: children);
+    final String flexWidgetName = properties.type.toString();
+    return Dialog(
```

Comment:
why is this a Dialog? We probably just want a vertically scrollable area.

Reply:
Right, this is still the old version, haven't put this into the tab yet